submissions/Neskazhuk/popup/index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600&display=swap" rel="stylesheet">
+    <link href="styles.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a class="link" href="#" tabindex="1">Hello</a></li>
+                <li class="nav-item"><a class="link" href="#" tabindex="2">World</a></li>
+                <li class="nav-item popup">
+                    <input id="popup-input-id" class="popup-input" type="checkbox" tabindex="3">
+                    <label for="popup-input-id" class="popup-btn">
+                        <img class="popup-icon" src="./icons/popup-button.png" alt="popup icon">
```

Comment:
Why did you leave this `alt`?)

Reply:
Missed)I'll fix it

Reply:
Looks like you forget to update the code and demo because I can't see changes)

Reply:
Now updated)